packages/nimble-components/src/table-column/base/cell-view/index.ts

```diff
@@ -47,11 +50,19 @@ export abstract class TableCellView<
         }
         this.delegatedEvents = this.column.columnInternals.delegatedEvents;
         this.delegatedEventHandler = (event: Event) => {
-            this.column?.dispatchEvent(
-                new CustomEvent<DelegatedEventEventDetails>('delegated-event', {
-                    detail: { originalEvent: event }
-                })
-            );
+            if (this.rowRecordId) {
+                this.column?.dispatchEvent(
+                    new CustomEvent<DelegatedEventEventDetails>(
+                        'delegated-event',
+                        {
+                            detail: {
+                                originalEvent: event,
+                                rowRecordId: this.rowRecordId
```

Comment:
What was the rationale in calling this `rowRecordId` rather than just `recordId`? I think we are pretty consistent throughout our code in calling this a `recordId`, so I think we should have a good justification for differing from that naming here.

Reply:
good catch!

Reply:
Mutated from my original name `rowId`. Agree it is better as just `recordId`.